components/landing/LandingActions.tsx

```diff
@@ -62,17 +65,17 @@ const LandingActions: StylableFC = ({ style, className }) => {
       <Button
         appearance="outlined"
         icon={<MaterialIcon icon="star" />}
-        onClick={() =>
-          plausible("Open Patch Notes", { props: { location: "Landing" } })
-        }
-        href="https://github.com/suankularb-wittayalai-school/mysk-frontend/pulls?q=is%3Apr+is%3Aclosed+base%3Amain+release+in%3Atitle"
-        // eslint-disable-next-line react/display-name
-        element={forwardRef((props, ref) => (
-          <a ref={ref} {...props} target="_blank" />
-        ))}
+        onClick={() => {
+          plausible("Open Patch Notes", { props: { location: "Landing" } });
+          setPatchNotesOpen(true);
+        }}
       >
         {t("new")}
       </Button>
+      <PatchNotesDialog
```

Comment:
From last time I haven’t added the closing feature. Now I see where the useState is being used.